dtests/src/test/scala/io/snappydata/hydra/SnappyHydraRunner.scala

```diff
@@ -65,7 +65,11 @@ class SnappyHydraRunner extends SnappyTestRunner {
     val c14 = "grep -v java.lang.reflect.InvocationTargetException"
     val c15 = "grep -v org.apache.spark.storage.ShuffleBlockFetcherIterator." +
         "throwFetchFailedException"
-    val command1 = c1 #| c2 #| c3 #| c4 #| c5 #| c6 #| c7 #| c8 #| c12 #| c13 #| c14 #| c15
+    val c16 = "grep -v java.lang.IllegalStateException"
+    val c17 = "grep -v com.gemstone.gemfire.distributed.LockServiceDestroyedException"
+    val c18 = "grep -v com.gemstone.gemfire.GemFireIOException"
+    val command1 = c1 #| c2 #| c3 #| c4 #| c5 #| c6 #| c7 #| c8 #| c12 #| c13 #| c14 #| c15 #|
+        c16 #| c17 #| c18
```

Comment:
These are not test specific. We always see these exception in logs, so these are expected list of exceptions. In the hydra tests, I am going to add the functionality to check for any exceptions and then fail the test if unexpected exception occurred. Currently this is not done, as I was not sure which exceptions should come under expected list considering HA failure list. Will be working on this after 1.0.